src/Illuminate/Support/Facades/Date.php

```diff
@@ -81,7 +81,7 @@
  * @method static \Illuminate\Support\Carbon today($tz = null)
  * @method static \Illuminate\Support\Carbon tomorrow($tz = null)
  * @method static void useMonthsOverflow($monthsOverflow = true)
- * @method static \Illuminate\Support\Carbon useStrictMode($strictModeEnabled = true)
+ * @method static void useStrictMode($strictModeEnabled = true)
```

Comment:
All the changes to the `Date` facade are unrelated to this PR. The underlying class was updated.